modules/news/news.model.js

```diff
@@ -0,0 +1,18 @@
+const newsType = `
+_id: ID!
+title: [Language!]
+text: [Language!]
+images: [PrimaryImage!]
+video: String!
```

Comment:
Not every piece of news will contain an image and I bet very few will contain a video

Reply:
fixed